packages/augment-api/src/interfaces/types-lookup.ts

```diff
@@ -1922,7 +1922,7 @@ declare module '@polkadot/types/lookup' {
     readonly asSr25519: SpCoreSr25519Signature;
     readonly isEcdsa: boolean;
     readonly asEcdsa: SpCoreEcdsaSignature;
-    readonly type: 'Ed25519' | 'Sr25519' | 'Ecdsa';
+    readonly type: 'ed25519' | 'sr25519' | 'ecdsa';
```

Comment:
These enum types unfortunately are dishonest now, which could be a real tricky trap for the ts user. If you interact with such an enum instance, the `type` string will still be sentence cased. This is because the types as used by the api instance are read directly from the chain's metadata and will therefore not respond to the changes made here.

Reply:
Yep, let’s undo that